website/blog/2025-10-15-introducing-atmos-auth-list.md

```
@@ -0,0 +1,197 @@
+# View specific identities
+atmos auth list --identities=admin,developer
+
+# Show all providers (no identities)
+atmos auth list --providers
+```
+
+### 🔗 Authentication Chain Visualization
+
+Understand complex authentication flows at a glance. Chains show the complete path from provider to target identity:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)